vite.config.ts

```diff
@@ -12,5 +14,8 @@ export default defineConfig({
     alias: {
       '@': fileURLToPath(new URL('./src', import.meta.url))
     }
+  },
+  define: {
+    'import.meta.env.PACKAGE_VERSION': JSON.stringify(version)
```

Comment:
What is this? `utils.makeEnvelope` already worked fine without meta env tricks as far as I know. Why is this necessary?